schemas/customAsset.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "customAsset.schema.json",
+  "title": "Custom Asset",
+  "description": "A Ready Player Me glTF binary validation schema for a custom asset.",
+  "type": "object",
+  "properties": {
+    "meshes": {
+      "type": "object",
+      "properties": {
+        "totalTriangleCount": {
+          "$ref": "meshTriangleCount.schema.json#/properties/custom"
+        },
+        "properties": {
+          "$comment": "inspect() creates a 'properties' object. Do not confuse with the 'properties' keyword.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "allOf": [
+              {
+                "$ref": "commonMesh.schema.json"
+              }
+            ],
+            "required": [
+              "mode",
+              "primitives",
+              "glPrimitives",
+              "vertices",
+              "indices",
+              "instances",
+              "size"
+            ]
+          },
+          "minItems": 1,
+          "maxItems": 20,
+          "errorMessage": {
+            "minItems": "Missing mesh! Full-body outfit asset must have at least 1 mesh.",
+            "maxItems": "Too many meshes! Full-body outfit asset must have no more than 4 meshes."
+          }
+        }
+      }
+    },
+    "textures": { "$ref": "commonTexture.schema.json#/$defs/fullPBR" }
```

Comment:
That definition needs textures to be square. I don't think we want to impose such restriction on custom assets.